### DISCUSSION TITLE
NTOPNG, is it in the roadmap to get integrated?

### DISCUSSION THREAD

**Discussion by TotieBash** [Security-Onion-Solutions/securityonion#4810] [Ideas] [answered]
I am looking for network flow top IP talker, top protocol, etc dashboard.  I see the zeek.conn Kibana dashboard as a good step forward but I wanted to see it in Grafana dashboard with top talkers and Mbps graph.  I love the SO Grafana dashboad btw so wanted something like that for network flows. I see that NTOPNG can push to influxdb so it looks like almost all of the dependencies are already in SO.  Anyone testing this? NTOPNG by it self is already a good product for looking at real time graphs but I feel like it lacks feature when graphing historical data so I like to see it in Grafana.

**Comment by weslambert** [comment, marked as answer]
ntopng is not on the roadmap at this time.
